common/test/acceptance/pages/studio/course_info.py

```diff
@@ -41,11 +41,9 @@ def is_update_button_enabled():
             """
             Checks if the New Update button is enabled
             """
-            update_button_prop = self.q(css='.new-update-button').attrs('disabled')[0]
-            if update_button_prop is not None:
-                return len(update_button_prop) == 0
-            else:
+            if self.q(css='.new-update-button').attrs('disabled')[0] is None:
```

Comment:
Will do, I was just running to tests with the Jenkins tester to verify it is working and not flaky.